corehq/pillows/mappings/domain_mapping.py

```diff
@@ -151,6 +151,7 @@
                                                    'section_title': {'type': 'string'}},
                                     'type': 'object'},
                 'full_downloads': {'type': 'long'},
+                'ga_opt_out': {'type': 'boolean'},
```

Comment:
I don't think this needs to be in ES unless there's a specific reason to add it.